.ci/.jenkins-schema.yml

```
@@ -0,0 +1,15 @@
+---
+agents:
+  - REPO: "apm-agent-dotnet"
+    SPEC_FILEPATH: "test/Elastic.Apm.Tests/TestResources/json-spec"
+  - REPO: "apm-agent-go"
+    SPEC_FILEPATH: "internal/apmschema/jsonschema"
+  - REPO: "apm-agent-java"
+    SPEC_FILEPATH: "apm-agent-core/src/test/resources/apm-server-schema/current"
+  - REPO: "apm-agent-nodejs"
```

@trentm  pointed to use https://github.com/elastic/apm-agent-nodejs/tree/master/test/integration/api-schema/apm-server-schema

```suggestion
  - REPO: "apm-agent-nodejs"
    SPEC_FILEPATH: "test/integration/api-schema/apm-server-schema"
```